src/app/styles/_media-mixin.scss

```diff
@@ -1,10 +1,11 @@
-@import "../../../node_modules/include-media/dist/include-media";
-@import "./variables";
+@import "~include-media/dist/include-media";
```

Comment:
it's unnerving seeing a `~` without a trailing slash. Is this correct?

Reply:
yes it is correct